core-web/libs/dot-primeng-theme-styles/src/scss/_variables.scss

```diff
@@ -1,13 +1,59 @@
 @use "sass:math";
 
-/*
-* Variable names convention:
-* element-property-state
-*/
-
-// COLORS
+// NEW COLORS WIP
```

Comment:
A few days ago you created

```
$brand-primary-100: #e9ebfc;
$brand-primary-100-hover: #d2d6f9;
```

Why don't remove those in favor of this new ones?